docs/_docs/settings.md

```diff
@@ -52,6 +52,8 @@ default.
 
 ![](/img/api-settings.png)
 
+You can opt-in to test new features with the **Feature Previews** option.
```

Comment:
Can you add a little more to this? Like what features we're testing and how this would affect the user. "You can switch back at any time" would be a good phrase to add too.

Reply:
I added more words :smile:

Reply:
Awesome